test/metabase/query_processor_test/parameters_test.clj

```diff
@@ -210,6 +210,43 @@
                              :target [:dimension [:template-tag "price"]]
                              :value  [1 2]}]}))))))
 
+(deftest params-in-comments-test
+  (testing "Params in SQL comments are ignored"
+    (testing "Single-line comments"
+      (mt/dataset airports
+                  (is (= {:query  "SELECT NAME FROM COUNTRY WHERE \"PUBLIC\".\"COUNTRY\".\"NAME\" IN ('US', 'MX') -- {{ignoreme}}"
+                          :params nil}
+                         (qp/compile-and-splice-parameters
+                          {:type       :native
+                           :native     {:query         "SELECT NAME FROM COUNTRY WHERE {{country}} -- {{ignoreme}}"
+                                        :template-tags {"country"
+                                                        {:name         "country"
+                                                         :display-name "Country"
+                                                         :type         :dimension
+                                                         :dimension    [:field (mt/id :country :name) nil]
+                                                         :widget-type  :category}}}
+                           :database   (mt/id)
+                           :parameters [{:type   :location/country
+                                         :target [:dimension [:template-tag "country"]]
+                                         :value  ["US" "MX"]}]})))))
```

Comment:
This should be indented 2 spaces since it's a macro with a `body`

https://guide.clojure.style/#body-indentation